flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/BufferedUpsertSinkFunctionTest.java

```diff
@@ -256,17 +273,17 @@ private BufferedUpsertSinkFunction createBufferedSink(MockedSinkFunction sinkFun
                         new int[] {keyIndices},
                         typeInformation,
                         BUFFER_FLUSH_MODE);
+        bufferedSinkFunction.getRuntimeContext().getExecutionConfig().enableObjectReuse();
```

Comment:
You already set object reuse in the underlying sink function, so don't need to set here again.